cmd/pulumi-test-language/providers/simple_invoke_provider.go

```diff
@@ -279,9 +283,9 @@ func (p *SimpleInvokeProvider) Check(
 		}, nil
 	}
 
-	if len(req.News) != 0 {
+	if len(req.News) > 1 {
```

Comment:
It's not required in `RequiredInputs` since the existing invoke tests didn't pass the prop, but it probably should be and it doesn't add too much noise to the other invoke tests.